src/razor/syntaxes/aspnetcorerazor.tmLanguage.json

```diff
@@ -64,6 +64,16 @@
         }
       ]
     },
+    "expression": {
```

Comment:
Nit: Perhaps a more specific name would be good here, just in case. Even "implicit-or-explicit-expression" would be good, and less overloaded than just "expression"

Reply:
That makes sense! I will push out another commit for that